com.brave.Brave.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.brave.Brave
+base: org.electronjs.Electron2.BaseApp
+base-version: '20.08'
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: brave
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--socket=wayland'
+  - '--socket=cups'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.freedesktop.GeoClue2'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.*'
+  - '--filesystem=host-etc'
+  - '--filesystem=xdg-documents'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-videos'
+modules:
+  - name: brave
+    buildsystem: simple
+    build-commands:
+      - 'install -Dm 755 brave.sh /app/bin/brave'
+      - 'install -Dm 755 apply_extra.sh /app/bin/apply_extra'
+      - 'install -Dm 755 stub_sandbox.sh /app/bin/stub_sandbox'
+      - 'install -Dm 644 -t /app/share/applications com.brave.Brave.desktop'
+      - 'install -Dm 644 -t /app/share/metainfo com.brave.Brave.metainfo.xml'
+      - |
+          for icon in 16 24 32 48 64 128 256; do
```

Comment:
You can download the [PublisherMediaAssets.zip](https://www.brave.com/static-assets/files/PublisherMediaAssets.zip) and use `../Logos/Brave/SVG/logo/primary/brave_lion.svg` instead of all the PNGs: `install -D --mode=0644 /path/to/brave_lion.svg /app/share/icons/hicolor/scalable/apps/com.brave.Brave.svg`.

If you want to make sure you always have the latest, you can [use the ZIP file as a source](https://github.com/flathub/com.jetbrains.IntelliJ-IDEA-Ultimate/blob/master/com.jetbrains.IntelliJ-IDEA-Ultimate.json#L27).

Reply:
I don't think I can because this will be used by `.desktop`. I'm not aware of any way to use `.svg`s with `.desktop`

Reply:
Feel free to unresolve this

Reply:
When you use the SVG, the location is different: `/app/share/icons/hicolor/scalable/apps/`. That alone is sufficient to convert and place automatically those PNGs during the build/packaging process.

Reply:
Installing the svg as well would be appreciated.